### PR TITLE
RK-643 updated throw_errors

### PR DESCRIPTION
**Pull request by BenTheKlein**
Do not merge since rooks are not published yet
@orishavit WDYT about the java docs? i added the other start function; should i delete it?

### CODE REVIEW

**Comment by LiranLast** [comment]
Ben, why would you delete that?

בתאריך יום ד׳, 19 בדצמ׳ 2018, 7:29, מאת Ben Klein <notifications@github.com

**Comment by BenTheKlein** [comment]
@LiranLast - since its not relevant to  `Rookout ETL Agent` anymore; just for agentless;
revert that change?

**Comment by BenTheKlein** [comment]
@LiranLast done;

File: docs/sdk-setup.md [hunk 1]
```diff
@@ -494,7 +496,7 @@ The `start` method is used to initialize the SDK in the background and recieves
 | `host` | `ROOKOUT_AGENT_HOST` | None | If you are using a Rookout ETL Agent, this is the hostname for it |
 | `port` | `ROOKOUT_AGENT_PORT` | None | If you are using a Rookout ETL Agent, this is the port for it |
 | `debug` | `ROOKOUT_DEBUG` | False | Set to `True` to increase log level to debug |
-| `silence_errors` | None | False | Set to `True` to throw if `start` fails |
+| `throw_errors` | None | False | Set to `True` to throw if `startWithExceptions` fails |
```
Comment (LiranLast) [inline docs/sdk-setup.md]: @BenTheKlein how does this line make sense? :/

**Comment by BenTheKlein** [comment]
@Rookeret73 should we write that when throw_errors is true no prints will be printed (except of the fork)

**Comment by BenTheKlein** [comment]
@LiranLast its ready - lets merge this

**Pull request merged**